test/parallel/test-crypto-dh.js

```diff
@@ -56,21 +59,29 @@ const secret3 = dh3.computeSecret(key2, 'hex', 'base64');
 
 assert.strictEqual(secret1, secret3);
 
+const wrongBlockLength =
+    new RegExp('^Error: error:0606506D:digital envelope' +
```

Comment:
@lpinca, looks like we really need this code style defined, the linter is not effective enough to solve this questions and the answer differs among collaborators